src/models/serverdata.cpp

```diff
@@ -295,12 +295,18 @@ void ServerData::setExitServerPublicKey(const QString& publicKey) {
   m_exitServerPublicKey = publicKey;
 }
 
-const Server ServerData::backupServer(const QString& currentPublicKey) const {
```

Comment:
Ideally, the backup servers should be selected according to the weighting as well....  but that is hard to do so I accept that this should be fine for fallback and recovery purposes. (See the mess of an algorithm in `Server::weightChooser()` for how it's done for the primary server) but a comment would be nice for our future selves.

Reply:
Philosophizing - I *almost* went that way, but this idea of additional backups isn't relevant to desktop. (We pass just the main server to the platform-specific controller, and then the mobile controller requests additional backup servers. And for desktop controllers, they get the main server and that's all they ever need - any server switches will come from the client.)